package.json

```diff
@@ -781,6 +781,11 @@
           "default": false,
           "description": "Do not show the Powershell Integrated Console banner on launch"
         },
+        "powershell.integratedConsole.cwd": {
```

Comment:
The cwd of the process will affect more than just the integrated console, it will affect completions as well for example. So it might be better to name this less specifically for the integrated console. I believe the original issue suggested `"powershell.cwd"`.